tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -77,6 +80,22 @@ public Operation operation(String name) {
   }
 
   /**
+   * Returns all operations that exist in the Graph.
+   */
+  public List<Operation> getOperations() {
```

Comment:
Ya, the specification doesn't mention it, but it is basically ubiquitous in the implementation of the collections in the JRE and so often i find people expect it.

Thanks for the feedback - l.